components/Task/Task.tsx

```diff
@@ -179,51 +182,53 @@ export default function Task({
         value={currentTask.description}
         style={{ height: showDescription ? textareaHeaight + "px" : "0px" }}

```

Comment:
literówka plus czemu nie jest to po prostu Textarea tylko input który udaje textarea?